validator/src/main/java/com/google/daq/mqtt/sequencer/Feature.java

```diff
@@ -0,0 +1,55 @@
+package com.google.daq.mqtt.sequencer;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Feature designation for line-item tests.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface Feature {
+  String IMPLICIT_CATEGORY = "";
+  Stage DEFAULT_STAGE = Stage.REQUIRED;
+  int DEFAULT_SCORE = 5;
+
+  /**
+   * Defines the category for this feature, as defined by a named attribute.
+   *
+   * @return feature category using named attribute
+   */
+  String category() default IMPLICIT_CATEGORY;
+
+  /**
+   * Defines the category for this feature, as defined by an implicit attribute.
+   *
+   * @return feature category using implicit argument
+   */
+  String value() default IMPLICIT_CATEGORY;
+
+  /**
+   * Default value is REQUIRED which should be the end-state for all tests.
+   *
+   * @return annotated release stage of this test
+   */
+  Stage stage() default Stage.REQUIRED;
+
+  /**
+   * Defines the feature score for this test, in AU.
+   *
+   * @return feature score value
+   */
+  int score() default DEFAULT_SCORE;
+
+  /**
+   * Enum of allowed stages.
+   */
```

Comment:
Is there a doc that pinpoints what each of these stages means to the UDMI project? For example ALPHA vs PREVIEW

Reply:
Also occurs to me, should "REQUIRED" be renamed "RELEASE"?

Reply:
I'd like to scope this PR at the mechanism for specifying the properties, but not the properties themselves (leave changing that to a separate PR). Once this PR is in place, it's easy to add/change the values themselves, and/or the properties associated with any given feature.

Reply:
If you change the enum to just one placeholder value I'm good with that. Then we can go back and edit the real values in place.

Reply:
How about two -- I'd like to keep default and non-default since the code/output is different in those two cases!  Updated, PTAL.

Reply:
Ok...as long as we can change REQUIRED to a different label if needed lgtm